core/templates/dev/head/css/oppia.css

```
@@ -3951,3 +3951,10 @@ span.oppia-issues-learner-action {
   width: 100%;
   border: 1px solid black;
 }
+
+/* To fix the shifting of tooltip when text is too long. */
+
+div.tooltip-inner {
```

I would be more specific here and go with 
```suggestion
.nav .tooltip-inner {
```